docs/en/sql-reference/functions/date-time-functions.md

```diff
@@ -180,6 +180,7 @@ Result:
 ## toYear
 
 Converts a date or date with time to a UInt16 number containing the year number (AD).
+Also accepts zero arguments and returns the current year at one of the moments of query analysis. The same as ‘toYear(now())’ or ‘toYear(today())’.
```

Comment:
"at one of the moments of query analysis" sounds obvious and weird. Yes, the wording was copypasted from other places in this file. It would be nice if you could omit that phrase in *all* places in this document.

Also, it's just fine to mention `toYear(now())` instead of `toYear(now()) or toYear(today())`.

--> "Also accepts zero arguments in which it returns the current year, i.e. `toYear(now())`"